examples/custom_procedure_example/main.py

```
@@ -1,38 +1,38 @@
-    for row in result["values"]:
-        print(row)
-
-
-if __name__ == "__main__":
+from datetime import datetime, timezone, timedelta
+
+# Import the high-level Python SDK
+from trufnetwork_sdk_py.client import TNClient
+
+# ---------------------------------------------------------------------------
+# Configuration – replace the values below with your own setup
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)